job_templates/sklearn_kmeans/config_fed_client.conf

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
Can we use one job template for all sklearn?
It should be possible with Chester's changes for job cli args addition / removal